addons/web_tour/static/src/js/tip.js

```diff
@@ -316,8 +316,29 @@ var Tip = Widget.extend({
                 of: this.$anchor,
                 collision: "none",
                 using: props => {
-                    this.el.style.setProperty('top', `${props.top}px`, 'important');
-                    this.el.style.setProperty('left', `${props.left}px`, 'important');
+                    const {top} = props;
+                    let {left} = props;
+                    const hasInnerText = this.$anchor.contents().first().text().trim().length;
+                    if (hasInnerText) {
+                        const textClone = this.$anchor.clone();
+                        const textRulerContainer = textClone.wrap('<div class="o_center_on_text">').parent();
```

Comment:
Make a more specific class name: start it with `o_tooltip_` like the other classes related to tips.